ramls/override-renewal-by-barcode-request.json

```diff
@@ -19,6 +19,11 @@
     "comment": {
       "description": "Comment to override action stored in loan history",
       "type": "string"
+    },
+    "overrideServicePointId": {
```

Comment:
Why is this named `overrideServicePointId` rather than `servicePointId`?

To me, the `override` prefix is redundant considering this is a request override aspects of a renewal.

Reply:
I can rename it.